### DISCUSSION TITLE
Incomplete label showing

### DISCUSSION THREAD

**Discussion by yaszin** [predict-idlab/plotly-resampler#309] [General]
Is there a way to show the complete trace label on the plots i.e : 
![image](https://github.com/predict-idlab/plotly-resampler/assets/67963519/d03865cf-ffed-441e-85de-19907e5aa0c4)

**Comment by jonasvdd** [comment]
Hi @yaszin, 

This is something caused by plotly's default behavior and not plotly-resampler. As described in [this issue](https://community.plotly.com/t/re-size-hover-info-box/955/12), this can be resolved by setting the `hoverlabel_namelength` attribute of a trace to -1.

```python
import plotly.graph_objects as go
from plotly_resampler import FigureResampler
import numpy as np

N = 1_000_000
x = np.arange(N)
y = np.sin(x / (N / 50))

fig1 = FigureResampler(go.Figure())
# NOTE: the hoverlabel_namelength attribute ensures that the full name is visible.
fig1.add_trace(
    go.Scatter(name="a trace", hoverlabel_namelength=-1),
    hf_x=x,
    hf_y=y,
)
fig1.add_trace(
    go.Scatter(name="another trace", hoverlabel_namelength=-1),
    hf_x=x,
    hf_y=y**2,
)
fig1.show_dash(mode="external", port=8001)
```
---

Without `hoverlabel_namelength=-1`:
![image](https://github.com/predict-idlab/plotly-resampler/assets/38005924/c89dd26c-0846-4246-9b18-2868ec476099)



With `hoverlabel_namelength=-1`:
![image](https://github.com/predict-idlab/plotly-resampler/assets/38005924/8c82b0ec-a6e7-48f0-bce1-b8cc5d40bf95)

I hope this helps you further!
Kind regards,
Jonas

**Reply by yaszin** [reply]
Thank you @jonasvdd this works !
